.env.prod

```
@@ -0,0 +1,122 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_DEBUG key should go before the APP_ENV key

(UnorderedKey)